docs/reference/notices.md

```diff
@@ -8,6 +8,30 @@ description: "Let's take a closer look at security notices, reporting vulnerabil
 
 Camunda publishes security notices after fixes are available.
 
+### Notice 14
+
+#### Publication date
+
+March 11th, 2025
```

Comment:
Can these both be listed under the same security notice? https://github.com/camunda/camunda-docs/pull/5149

Reply:
They are separate things in separate components. I assume we should have different security notices for these?